compiler/testData/ir/irText/expressions/vararg.txt

```diff
@@ -12,12 +12,10 @@ FILE fqName:<root> fileName:/vararg.kt
   PROPERTY name:test2 visibility:public modality:FINAL [val]
     FIELD PROPERTY_BACKING_FIELD name:test2 type:kotlin.Array<kotlin.String> visibility:private [final,static]
       EXPRESSION_BODY
-        CALL 'public final fun arrayOf <T> (vararg elements: T of kotlin.arrayOf): kotlin.Array<T of kotlin.arrayOf> [inline] declared in kotlin' type=kotlin.Array<kotlin.String> origin=null
-          <T>: kotlin.String
-          elements: VARARG type=kotlin.Array<out kotlin.String> varargElementType=kotlin.String
-            CONST String type=kotlin.String value="1"
-            CONST String type=kotlin.String value="2"
-            CONST String type=kotlin.String value="3"
+        VARARG type=kotlin.Array<kotlin.String> varargElementType=kotlin.String
```

Comment:
Please pay attention to the fact that this file was "FIR_IDENTICAL", and it means that FE 1.0 (aka old front-end) does not transform regular (not inside annotation arguments) `arrayOf` calls this way you suggest.

Reply:
Sorry, didn't noticed this one doesn't have `fir` suffix in the file name. Reverted with additional guard regarding when to transform `arrayOf` calls.